nixos/tests/nzbget.nix

```diff
@@ -0,0 +1,22 @@
+import ./make-test.nix ({ pkgs, ...} : {
+  name = "nzbget";
+  meta = with pkgs.stdenv.lib.maintainers; {
+    maintainers = [ aanderse flokli ];
+  };
+
+  nodes = {
+    server = { ... }: {
+      services.nzbget.enable = true;
```

Comment:
@flokli This doesn't properly evaluate on the bot because `unrar` requires `nixpkgs.config.allowUnfree = true;` :sob:

Reply:
Booooh!